src/frontend/src/common/components/library/data_table/components/HeaderRow/index.tsx

```diff
@@ -0,0 +1,60 @@
+import { isEqual } from "lodash";
+import React from "react";
+import { TableHeader } from "./components/TableHeader";
+import { HeaderCheckbox, StyledHeaderRow } from "./style";
+
+interface Props {
+  handleHeaderCheckboxClick(): void;
+  handleSortClick(key: string[]): void;
+  headers: Header[];
+  isHeaderChecked: boolean;
+  isHeaderIndeterminant: boolean;
+  isSortedAscending: boolean;
+  shouldShowCheckboxes: boolean;
+  sortColKey: string[];
```

Comment:
i think renaming this to `sortColKeys` make more sense since we have multiple columns to sort on?

Reply:
Yea, I was mixed on this. The reason I kept it as singular was because the thing we're comparing it to (`sortKey`) is also an array specifying potentially multiple strings, but was named singular.

Reply:
i would also rename that to sortKeys 😆, but too late now